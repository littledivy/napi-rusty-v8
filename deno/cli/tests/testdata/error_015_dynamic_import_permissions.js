(async () => {
  await import("http://localhost:4545/subdir/mod4.js");
})();
