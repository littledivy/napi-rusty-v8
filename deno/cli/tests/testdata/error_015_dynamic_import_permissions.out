error: Uncaught (in promise) TypeError: Requires net access to "localhost:4545", run again with the --allow-net flag
  await import("http://localhost:4545/subdir/mod4.js");
  ^
    at async file://[WILDCARD]/error_015_dynamic_import_permissions.js:2:3
